README.md

```diff
@@ -933,16 +1003,176 @@ SUB COMMANDS:
     dump          Dump entire OTP (RP2350 only)
     permissions   Set the OTP access permissions (RP2350 only)
     white-label   Set the white labelling values in OTP (RP2350 only)
-
 ```
 
 ### set/get
 
 These commands will set/get specific rows of OTP. By default, they will write/read all redundant rows, but this can be overridden with the `-c` argument
 
+```text
+$ picotool help otp get
+OTP GET:
+    Get the value of one or more OTP registers/fields
+
+SYNOPSIS:
+    picotool otp get [-c <copies>] [-r] [-e] [-n] [-i <filename>] [device-selection] [-z] [<selector>..]
+
+OPTIONS:
+    Row/field options
+        -c <copies>
+            Read multiple redundant values
+        -r, --raw
+            Get raw 24 bit values
```

Comment:
Small nit, but in general I think "24-bit values" looks better than "24 bit values".